doc_source/serverless-controlling-access-to-apis-lambda-authorizer.md

```diff
@@ -135,4 +135,12 @@ Resources:
       CodeUri: ./src
       Handler: authorizer.handler
       Runtime: nodejs12.x
-```
\ No newline at end of file
+
+  MyAuthFunctionApiGatewayInvokePermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      Action: lambda:InvokeFunction
+      FunctionName: !GetAtt MyAuthFunction.Arn
+      Principal: apigateway.amazonaws.com
+      SourceArn: !Sub "arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${MyApi}/*"
```

Comment:
On ARNs: I'm curious about the difference  between:
```yml
!Sub "arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${MyApi}/*"
```
and
```yml
!Sub "arn:aws:execute-api:${AWS::Region}:${AWS::AccountId}:${MyApi}/authorizers/*"
```
Intuitively it seems adding the suffix `/authorizers/*` only permits Lambda Authorizers.
If that is a correct understanding I would propose updating these changes to reflect that. Why?  principle of least privilege!

Reply:
Seems right. This is the best I can come up with as far as documentation that you are right. Although probably want to add the authorizer id if that is the case.

https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-lambda-authorizer.html#http-api-lambda-authorizer.example-create